docs/installation/using-with-jest.md

```diff
@@ -4,8 +4,24 @@ Jest is a test runner developed by Facebook, aiming to deliver a battery-include
 
 <div class="vueschool"><a href="https://vueschool.io/courses/learn-how-to-test-vuejs-components?friend=vuejs" target="_blank" rel="sponsored noopener" title="Learn how to use Jest and Vue Test Utils to test Single File Components with Vue School">Learn how to use Jest to test Single File Components with Vue School</a></div>
 
+### Installation with Vue CLI (recommended)
+
 If you are using the Vue CLI to build your project, you can use the plugin [cli-plugin-unit-jest](https://cli.vuejs.org/core-plugins/unit-jest.html) to run Jest tests.
 
+```bash
+$ vue add unit-jest
+```
+
+The plugin pulls all required dependencies (including jest), creates a `jest.config.js` file with sensible defaults, and generates a sample test suite.
```

Comment:
Looks great! Could we drop a link to the plugin documentation?

```suggestion
The plugin ([docs](https://cli.vuejs.org/core-plugins/unit-jest.html)) pulls all required dependencies (including jest), creates a `jest.config.js` file with sensible defaults, and generates a sample test suite.
```

Reply:
There is already a link to the official docs on line 9. I don't mind adding a second one if you think it's needed, but I just want to make sure that is indeed your intent :)

Reply:
yeah, totally missed that 👍